apps/backend/src/modules/categories/category.service.ts

```diff
@@ -38,12 +41,18 @@ class CategoryService implements Service {
 		};
 	}
 
-	public async create(payload: CategoryRequestDto): Promise<CategoryDto> {
+	private parseEntityToDto(categoryEntity: CategoryEntity): CategoryDto {
```

Comment:
Try to reuse what we already have with scores. In your case scores will be an empty array
Same for all other methods

Reply:
Could you please clarify what you mean by 'reuse what we already have with scores'? I'm not entirely sure what to implement. Do I need rafactor the repository too?